notebook/static/notebook/less/textcell.less

```diff
@@ -48,6 +48,10 @@ h1,h2,h3,h4,h5,h6 {
     display:none;
 }
 
+.text_cell .droppable{
```

Comment:
can you target the `.input_area` in particular, which already have a border, so this will avoid the change of size. And you can set only border-color.